core/src/test/java/org/apache/brooklyn/core/mgmt/internal/TestEntityWithEffectors.java

```diff
@@ -21,15 +21,77 @@
 import org.apache.brooklyn.api.entity.ImplementedBy;
 import org.apache.brooklyn.core.annotation.Effector;
 import org.apache.brooklyn.core.annotation.EffectorParam;
+import org.apache.brooklyn.core.effector.MethodEffector;
 import org.apache.brooklyn.core.test.entity.TestEntity;
 
+/**
+ * Entity for testing that secret effector parameters are:
+ * <ul>
+ *   <li>excluded from the activities view
+ *   <li>not logged
+ *   <li>masked out in the UI
+ * </ul>
+ * Of those, only the first is unit-tested.
+ * 
+ * To test manually...
+ * 
+ * Configure logback to log everything at trace:
+ * <pre>
+ * {@code
+ * <configuration>
+ *     <include resource="logback-main.xml"/>
+ *     <logger name="org.apache.brooklyn" level="TRACE"/>
+ *     <logger name="brooklyn" level="TRACE"/>
+ * </configuration>
+ * }
+ * </pre>
+ * 
+ * Run Brooklyn with the above log configuration file:
+ * <pre>
+ * {@code
+ * export JAVA_OPTS="-Xms256m -Xmx1g -XX:MaxPermSize=256m -Dlogback.configurationFile=/path/to/logback-trace.xml"
+ * ./bin/brooklyn launch --persist auto --persistenceDir /path/to/persistedState
+ * }
+ * </pre>
+ * 
+ * Deploy the blueprint below:
+ * <pre>
+ * {@code
+ * services:
+ * - type: org.apache.brooklyn.core.mgmt.internal.TestEntityWithEffectors
```

Comment:
The way I built it was to export TestEntityWithEffectors*.java as a jar from my IDE, and add it to ./lib/patch of a pre-installed Brooklyn. That didn't require adding the pom.

Which pom did you add that to? Maybe we should add something more general like "You'll have to ensure this is on the classpath of your brooklyn server, before testing". I'll update my PR accordingly.